### PR TITLE
Angularjs removal

### LINKED ISSUE #6991: Remove AngularJS 1 from main generator and move to blueprint
We have decided to remove support for AngularJS version 1.x from the core generator for next major version.
The feature will be moved into a blueprint(beta feature) and will become an official blueprint which will be maintained independently of the core.

Team discussion here https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/jhipster-dev/v5v4xgpu6Kc

### PR DESCRIPTION
**Pull request by deepu105**
fix #6991 

Removing AngularJS 1.x from the main generator. This will be published as a blueprint `generator-jhipster-angularjs` under jhipster org

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
I will update hipster-labs and remove all ng1 builds

**Comment by deepu105** [comment]
@pascalgrimaud yes we can move the ng1 builds into the blueprint

**Comment by pascalgrimaud** [comment]
I can help to do the build on this.
Ping me as soon as the repo is created

**Comment by pascalgrimaud** [comment]
Yes I saw it today, I restarted a lot of builds and it passed.
Travis not very stable today.

**Comment by amirensit** [comment]
When will be jhipster 5 released ?

**Comment by jdubois** [comment]
@amirensit we don't commit on any date, it's done when it's done

**Comment by amirensit** [comment]
@jdubois ok.

**Comment by deepu105** [comment]
@pascalgrimaud I have no idea why one of the build is failing. can you take look. I have restarted it several times already

**Comment by jdubois** [comment]
@deepu105 be careful when push this to your repo, I made this small change https://github.com/jhipster/generator-jhipster/commit/1fbbb901d27e8ccac65e7a49b6764fb3e238fdf5 that you need to cherry pick

**Comment by jdubois** [comment]
@deepu105 @pascalgrimaud the build is failing because there's a test using AngularJS here - I'm having a look

**Comment by pascalgrimaud** [comment]
Thanks, I can't have a look right now, as I'm still at work :)

**Comment by jdubois** [comment]
Well, I was wrong, this is using Angular 5.... And I have no idea what's happening, we have this error (which I have already seen before):

 ```
2018-02-06 07:53:16.461 DEBUG 9201 --- [           main] i.g.j.c.apidoc.SwaggerConfiguration      : Started Swagger in 12 ms
* Rebuilt URL to: http://localhost:8080/
* Hostname was NOT found in DNS cache
```

I'm restarting the build to see

File: generators/server/templates/_pom.xml [hunk 1]
```diff
@@ -1221,7 +1206,7 @@
                         <configuration>
                             <failOnMissingWebXml>false</failOnMissingWebXml>
                             <%_ if (!skipClient) { _%>
-                            <warSourceDirectory><% if (clientFramework !== 'angular1') { %><%= CLIENT_DIST_DIR %><% } else { %><%= CLIENT_MAIN_SRC_DIR %><% } %></warSourceDirectory>
+                            <warSourceDirectory><%= CLIENT_MAIN_SRC_DIR %></warSourceDirectory>
```
Comment (pascalgrimaud) [inline generators/server/templates/_pom.xml]: @deepu105 : I think the correct value should be `CLIENT_DIST_DIR`, according to the condition `!== 'angular1'`

File: generators/server/templates/src/main/java/package/config/_WebConfigurer.java [hunk 2]
```diff
@@ -127,15 +127,11 @@ public void customize(ConfigurableEmbeddedServletContainer container) {
     private void setLocationForStaticAssets(ConfigurableEmbeddedServletContainer container) {
         File root;
         String prefixPath = resolvePathPrefix();
-        <%_ if (clientFramework !== 'angular1') { _%>
-        root = new File(prefixPath + "<%= CLIENT_DIST_DIR %>");
```
Comment (pascalgrimaud) [inline generators/server/templates/src/main/java/package/config/_WebConfigurer.java]: Same comment here. I think we should keep this part, instead.

**Comment by pascalgrimaud** [comment]
@deepu105 :
- you have some merge conflicts
- I added 2 comments in your commits, the build failed because of that
- why the other ngx builds didn't fail ?
    - prod + protractor -> OK
    - dev + protractor -> KO, there is only 1 build which is ngx-mariadb-oauth2-sass-infinispan
    - the other ngx don't have protractor, that's the reason

**Comment by pascalgrimaud** [comment]
Sorry, I don't have time to push directly to your branch, to fix it
Too much work these last weeks, so too tired when back at home

**Comment by deepu105** [comment]
No worries thanks for taking a look. I'll fix them soon


Thanks & Regards,
Deepu

**Comment by deepu105** [comment]
or soonish :P


Thanks & Regards,
Deepu

**Comment by riaprocoumai** [comment]
generator-jhipster-angularjs not available as blueprint
whats name of angularJs blue print?

**Pull request merged**